gulpfile.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+/* eslint no-process-exit:0 */
+
+// load third party modules
+var del      = require( 'del' );
+var gulp     = require( 'gulp' );
+var mocha    = require( 'gulp-mocha' );
+var istanbul = require( 'gulp-istanbul' );
+
+var paths = {
+	'cover' : [
+
+		// Include everything to be covered
+		'**/*.js',
+
+		// Exclude files
+		'!gulpfile.js',
+
+		// Exclude directories that are not code
+		'!config/**',
+		'!instrumented/**',
+		'!node_modules/**',
+		'!test/**'
+	],
+
+	'test'     : [ 'test/**/*.js' ],
+	'coverage' : 'instrumented'
+};
+
+var covEnforcerOpts = {
```

Comment:
Let's use this package for enforcement options: https://www.npmjs.com/package/sinet-istanbul-coverage-enforcement